openapi/integrator/wallet/client.yaml

```diff
@@ -187,7 +187,7 @@ paths:
           $ref: '../components.yaml#/components/responses/500Error'
         '503':
           $ref: '../components.yaml#/components/responses/503Error'
-  /payments/authentication:
+  /wallet/payments/authentication:
```

Comment:
I guess you didn't make it like that, but isn't it it a bit weird to have two different path roots in the same yaml?

Reply:
I guess it will be better after payment-tokens will be adjusted with the same /wallet prefix, too. I just didn't want to change it now as we have a separate task for enrolment and trying to not go too much over my scope. But can be done, if there's a problem with leaving that one for later.